src/integrate/add_metadata/config.vsh.yml

```diff
@@ -0,0 +1,68 @@
+functionality:
+  name: add_metadata
+  namespace: "integrate"
+  description: "Add a csv containing metadata to the .obs or .var field of a mudata file."
+  authors:
+    - name: Dries Schaumont
+      email: dries@data-intuitive.com
+      roles: [ author ]
+      props: { account: DriesSchaumont, orcid: "0000-0002-4389-0440" }
+  arguments:
```

Comment:
* I'd rename --input to --input_h5mu, and --csv to --input_csv.
* I'd also use argument groups to split up the arguments somewhat
* I'd remove the `sample_id` column. If the obs_key or the var_key does not exist in .obs or .var respectively, throw an error.

Proposed argument groups:
```yaml
argument_groups:
  - name: "MuData input"
    arguments:
      - name: --input_h5mu
      - name: --modality
      - name: --obs_key
      - name: --var_key # must be mutually exclusive with obs_column
  - name: "CSV input"
    arguments:
      - name: --input_csv
      - name: --csv_key
   - name: Outputs
     arguments:
      - name: --output